CHANGELOG.md

```diff
@@ -39,6 +39,8 @@ All notable changes to this project will be documented in this file
 * changed function name from `vehicle` to `vehicleName` in vehicle module
 * changed function name from `timezone` to `timezoneRandom` in date module
 * deleted function `commonFileType` from `System` module, use `system.fileType` instead
+* changed function name from `number` (by format) to `phoneNumberByFormat` in phone module
```

Comment:
Please create a release v3 notes and place it there, you could also add a note we changed all files to snake_case